tests/Browser/UserTest.php

```diff
@@ -0,0 +1,204 @@
+     * Admin redirect to page edit user
+     * case 1-3
+     *
+     * @return void
+     */
+    public function testClickMenuToPageEdit()
+    {
+        $admin = $this->accountAdmin();
+        $data = $this->getDataTest();
+
+        $this->browse(function (Browser $browser) use ($admin, $data) {
+            $browser->loginAs($admin)
+                ->visit($data['path']['listUser'])
+                ->click('@editUser2')
+                ->assertPathIs('/admin/user/edit/2');
+        });
+    }
+
+    /**
+     * Admin show popup delete user
+     * case 1-4
+     *
+     * @return void
+     */
+    public function testPopupDeleteUser()
+    {
+        $admin = $this->accountAdmin();
+        $data = $this->getDataTest();
+
+        $this->browse(function (Browser $browser) use ($admin, $data) {
+            $browser->loginAs($admin)
+                ->visit($data['path']['listUser'])
+                ->click('@deleteModal2')
+                ->waitFor('#myModal2')
+                ->assertSee('Are you sure you want to delete');
+        });
+    }
+
+    /**
+     * Checkbox no check and not submit
+     * case 2-12
+     *
+     * @return void
+     */
+    public function testNotSubmitWithCheckboxAllNoCheck()
+    {
+        $admin = $this->accountAdmin();
+        $data = $this->getDataTest();
+
+        $this->browse(function (Browser $browser) use ($admin, $data) {
+            $browser->loginAs($admin)
+                ->visit($data['path']['listUser'])
+                ->click('@editUser2')
+                ->check('can_see', null)
+                ->check('can_delete', null)
+                ->assertPathIs('/admin/user/edit/2');
+        });
+    }
+
+    /**
+     * Checkbox checked but not submit
+     * case 2-13
+     *
+     * @return void
+     */
+    public function testNotSubmitWithCheckboxAll()
+    {
+        $admin = $this->accountAdmin();
+        $data = $this->getDataTest();
+
+        $this->browse(function (Browser $browser) use ($admin, $data) {
+            $browser->loginAs($admin)
+                ->visit($data['path']['listUser'])
+                ->click('@editUser2')
+                ->check('can_see')
+                ->check('can_delete')
+                ->assertPathIs('/admin/user/edit/2');
+        });
+    }
+
+    /**
+     * Admin delete user, press key Yes
+     * case 2-14
+     *
+     * @return void
+     */
+    public function testDeleteUser()
+    {
+        $admin = $this->accountAdmin();
+        $data = $this->getDataTest();
+
+        $this->browse(function (Browser $browser) use ($admin, $data) {
+            $browser->loginAs($admin)
+                ->visit($data['path']['listUser'])
+                ->click('@deleteModal2')
+                ->waitFor('#myModal2')
+                ->assertSee('Are you sure you want to delete')
+                ->press('Yes')
+                ->waitForLocation($data['path']['listUser'])
+                ->assertSee('Delete account success !');
```

Comment:
'Account deleted > show message success' , this is expected in test case , this arrert only tell me half of the info 'show message'

Reply:
I added case message "Account deleted"